makefiles/Makefile.inc

```diff
@@ -2,13 +2,7 @@
 # Licensed under the Revised BSD License, see LICENSE for details.
 # SPDX-License-Identifier: BSD-3-Clause
 
-COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
-ifeq ($(COCOTB_CONFIG_CMD),)
-    $(error 'cocotb-config not found! Cocotb is not installed (or is installed but not on the PATH). \
-             Please refer to https://cocotb.readthedocs.io/en/latest/quickstart.html#installing-cocotb \
-             for installation instructions.')
-endif
-
-$(error 'Deprecated cocotb file structure detected. \
-         Please refer to https://cocotb.readthedocs.io/en/latest/quickstart.html#creating-a-makefile \
-         for instructions.')
+$(error Outdated use of cocotb detected. \
+        Please install cocotb, and modify your makefile to \
+        "include $$(shell cocotb-config --makefile)/Makefile.sim" instead. \
+        Please refer to the documentation at https://docs.cocotb.org.)
```

Comment:
Are you sure we don't want to point specifically to the installation page?

Reply:
```suggestion
        Please refer to the documentation at https://docs.cocotb.org/en/latest/install.html )
```

Reply:
Because users need to look at two pages: the install page for how to install cocotb, and the quickstart page for how to fix their Makefile.